cosmos.config.json

```diff
@@ -1,7 +1,7 @@
 {
-  "staticPath": "public",
-  "watchDirs": ["src"],
+  "watchDirs": ["src/lib"],
```

Comment:
Change this to `src`, or add `src` to it, as we'll now be mixing the codebases while we implement hooks

Reply:
yeeah, was wondering about this one